src/Resizetizer/src/WindowIconGeneratorTask.cs

```diff
@@ -68,19 +79,41 @@ public static void SetWindowIcon(this global::Microsoft.UI.Xaml.Window window)
 			global::Microsoft.UI.Windowing.AppWindow appWindow =
 				global::Microsoft.UI.Windowing.AppWindow.GetFromWindowId(windowId);
 			appWindow.SetIcon(""{iconName}.ico"");
+			appWindow.Title = ""{windowTitle}"";
 #endif
 		}}
 	}}
 }}";
 
-		if(!Directory.Exists(IntermediateOutputDirectory))
-		{
-			Directory.CreateDirectory(IntermediateOutputDirectory);
-		}
-
 		var item = new TaskItem(Path.Combine(IntermediateOutputDirectory, FileName));
 		File.WriteAllText(item.ItemSpec, code);
 		GeneratedClass = new [] { item };
 		return true;
 	}
+
+	string GetDisplayName()
```

Comment:
Methods require accessor
```suggestion
	private string GetDisplayName()
```